tutorials/howtos/trim-index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "How can I remove pieces from an index listing?"
```

Comment:
This is great! You should specifically mention what an index listing is: a listing of pieces on a page powered by a module that extends apostrophe-pieces-pages, or apostrophe-blog-pages, apostrophe-events-pages, etc. which are derived from that.